### ISSUE TITLE
Я наверное тупой

### ISSUE THREAD

**Issue by goodandrewsoft** [Infocatcher/Custom_Buttons#46]
https://addons.mozilla.org/addon/custom-buttons/
Но по этой ссылке расширение недоступно.

Где скачать последнюю версию для браузера Firefox 66.0 64 bit ?

**Comment by Infocatcher** [comment]
Да, разработчики Firefox неплохо справляются с уничтожением хороших расширений.

Исправленные версии и распорки для их запуска периодически выкладывает <a href="https://forum.mozilla-russia.org/search.php?action=show_user&user_id=71856">Dumby</a> (за что ему огромное спасибо).
И, судя по периодическим вопросам насчет кода кнопок, кто-то эти версии даже успешно запускает.
В итоге расширение поддерживает сообщество в лице полутора разработчиков, а пользователи или просто молча используют результаты, или вовсе <a href="https://forum.mozilla-russia.org/viewtopic.php?id=70326">копируют</a> их без указания авторства.

Вот самая свежая версия на данный момент:
https://forum.mozilla-russia.org/viewtopic.php?pid=767960#p767960
Вот пояснения:
https://forum.mozilla-russia.org/viewtopic.php?pid=764797#p764797

В папке с firefox.exe надо создать два файла (в кодировке UTF-8 без BOM):

/config.js
```js
// Скопировать соответствующий код из https://forum.mozilla-russia.org/viewtopic.php?pid=767960#p767960
```
/defaults/pref/config-prefs.js
```js
pref("general.config.obscure_value", 0);
pref("general.config.filename", "config.js");
pref("general.config.sandbox_enabled", false);
```

После перезапуска установить custom_buttons-0.0.7.0.0.3-fx-paxmod.xpi и перезапустить браузер.

**Comment by goodandrewsoft** [comment]
Спасибо большое.
А для чего предназначен custom_buttons-0.0.7.0.0.3-fx-bootstrap.xpi и bootstrap-loader.js ?

**Comment by Infocatcher** [comment]
Это альтернативный вариант.
*-paxmod.xpi – это WebExtension (которые которые браузер понимает сам), но с добавлением реализации нового API (чтобы воспользоваться низкоуровневыми возможностям, которые недоступны для обычных WE-расширений).
*-bootstrap.xpi – в формате старых расширений без перезапуска, код для запуска которых уже удалили из браузера, поэтому нужно добавить в config.js дополнительный код из bootstrap-loader.js (дописать в конец файла), чтобы вернуть их поддержку.

**Comment by goodandrewsoft** [comment]
Замечательно. Спасибо за достойные ответы, и за расширение extended link properties.
Я вас знаю давно уже, с форума https://forum.mozilla-russia.org/viewtopic.php?id=9591 )
